kie-server-parent/kie-server-services/kie-server-services-jbpm/src/test/java/org/kie/server/services/jbpm/QueryDataServiceBaseTest.java

```diff
@@ -0,0 +1,54 @@
+package org.kie.server.services.jbpm;
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import org.jbpm.services.api.model.UserTaskInstanceWithPotOwnerDesc;
+import org.jbpm.services.api.query.QueryMapperRegistry;
+import org.jbpm.services.api.query.QueryResultMapper;
+import org.jbpm.services.api.query.QueryService;
+import org.junit.Test;
+import org.kie.server.api.KieServerEnvironment;
+import org.kie.server.services.api.KieServerRegistry;
+import org.kie.server.services.impl.KieServerRegistryImpl;
+import org.kie.server.services.impl.storage.file.KieServerStateFileRepository;
+import org.mockito.Mockito;
+
+
+public class QueryDataServiceBaseTest {
```

Comment:
instead of this test (or in addition to it) I would add new test that actually test entire flow here https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java

Reply:
I've added two tests more